QUEST_LV_0100.tsv

```diff
@@ -6600,7 +6600,7 @@ QUEST_LV_0100_20150323_006599	Morkus Jonas says you can listen to the next story
 QUEST_LV_0100_20150323_006600	Go and meet Rolandas Jonas in Akmens Ridge.
 QUEST_LV_0100_20150323_006601	{memo X}Goddess Saule says Bramble of Kvailas Forest is in Thorn Forest. Pass through Gate Route and Spine Heart Dale to Kvailas Forest to find the revelation. 
 QUEST_LV_0100_20150323_006602	Kvailas Forest
-QUEST_LV_0100_20150323_006603	Move to Gele Plateau
+QUEST_LV_0100_20150323_006603	Travel to Gele Plateau
```

Comment:
The Paladin Master is waiting for you in Gele Plateau, after you've received the revelation in the mines.  You need to travel across all the maps from the Miners' Village to Gele Plateau and meet him.